### PR TITLE
chore: Update fonts for production (with corrections)

### PR DESCRIPTION
**Pull request by catheraaine**
Add font files to be used in production by Typography.com

Update Grunt Copy to properly copy files to `dist`

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -15,8 +15,12 @@ GEM
     sass (3.2.19)
 
 PLATFORMS
+  java
```
Comment (cromwellryan) [inline Gemfile.lock]: Super surprised to see this. What ruby are you using?

File: grunt/contrib-copy.js [hunk 2]
```diff
@@ -4,8 +4,7 @@ module.exports = function(grunt) {
       files: [
         {expand: true, cwd: "pages/", src: "*.html", dest: "dist/"},
         {expand: true, cwd: "img/", src: "*", dest: "dist/img"},
-        {expand: true, cwd: "Fonts", src: "*", dest: "dist/Fonts"}
-      ]
+        {expand: true, cwd: "fonts/", src: "**/*", dest: "dist/fonts"}      ]
```
Comment (cromwellryan) [inline grunt/contrib-copy.js]: Can we fix that line break?

**Comment by cromwellryan** [comment]
Can you lower case the `Chore` in your commit message?

File: Gemfile.lock (hunk 1 above)
Comment (catheraaine) [inline Gemfile.lock]: my RVM defaults to jruby

**Pull request merged**